src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+form {
+  margin-top: 20px;
+}
+
+.search {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.big_s {
+  height: 70px;
+}
+
+.small_s {
+  height: 42px;
+}
+
+.text {
+  color: rgba(61, 78, 97, 1);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.big {
+  height: 65px;
+  background-position: 20px;
+  font-size: 16px;
+  padding-left: 62px;
+  border: none;
+}
+
+.small {
+  font-size: 14px;
+  line-height: 19px;
+  height: 16px;
+  padding: 11px 33px;
+  background-position: 13px 49%;
+  background-size: 11px;
+  border: none;
+}
+
+.big_s:hover,
+.small_s:hover {
+  box-shadow: 1px 1px 8px 1px rgba(61, 78, 97, 0.1);
+}
```

Comment:
Try to place selectors with pseudo-classes right after general selectors. This should be placed right after  `big_s` selector.